controllers/machine_controller_test.go

```diff
@@ -597,8 +597,9 @@ func TestRemoveMachineFinalizerAfterDeleteReconcile(t *testing.T) {
 	_, err := mr.Reconcile(reconcile.Request{NamespacedName: key})
 	g.Expect(err).ToNot(HaveOccurred())
 
-	g.Expect(mr.Client.Get(ctx, key, m)).To(Succeed())
-	g.Expect(m.ObjectMeta.Finalizers).To(Equal([]string{metav1.FinalizerDeleteDependents}))
+	var actual clusterv1.Machine
+	g.Expect(mr.Client.Get(ctx, key, &actual)).To(Succeed())
+	g.Expect(len(actual.ObjectMeta.Finalizers)).To(Equal(0))
```

Comment:
Using the gomega `BeEmpty()` matcher would result in a better error message if this assertion were to fail
```suggestion
	g.Expect(actual.ObjectMeta.Finalizers).To(BeEmpty())
```

Reply:
Thank you for your advice @detiber 👍